api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -121,6 +121,30 @@ paths:
           description: Resource not found.
           schema:
             $ref: "#/definitions/Error"
+  /thyra/plugin-manager/{resource}:
+    get:
+      operationId: thyraPluginManager
+      produces:
+        - application/json
+        - text/javascript
+        - text/html
+        - text/css
+        - text/webp
+        - image/png
+      parameters:
+        - in: path
+          name: resource
+          type: string
+          default: index.html
+          required: true
```

Comment:
Not sure you can:
- define a default value if parameter is in path
- require is true by default for parameter in path

What is the output of swagger when you ask it to validate your swagger file ?

Reply:
Honestly it's a copy pasta from all of others thyra/**plugins**/{resources}

And it works well on them.

![image](https://user-images.githubusercontent.com/10938862/218123875-8d1c65ed-fe9f-485b-8db7-d479b6c2574a.png)

![image](https://user-images.githubusercontent.com/10938862/218123905-b192d017-790e-4be2-bed7-71ef19e3cff2.png)

I didn't re-invent the wheel and i think go generate don't print any error
And swagger validate print this :

![image](https://user-images.githubusercontent.com/10938862/218124312-83891d86-e731-4408-9fe4-7c90054e0115.png)

Reply:
i tried to delete the default value. and go validate still show this

Reply:
i delete the default value. but i don't know if it breaks something else because this was not related to my subject